alvr/dashboard/src/dashboard/components/new_version_popup.rs

```diff
@@ -4,6 +4,11 @@ use alvr_gui_common::ModalButton;
 use alvr_packets::{PathValuePair, ServerRequest};
 use eframe::egui::{self, Context, OpenUrl, Ui};
 
+pub enum CloseAction {
+    Close,
+    CloseWithRequest(ServerRequest),
+}
```

Comment:
I opted for this enum instead of Option to be more clear. Nested Options is never a good idea.